frontend/src/index.tsx

```diff
@@ -13,6 +13,16 @@ import { loadPostHogJS } from './loadPostHogJS'
 loadPostHogJS()
 initKea()
 
+// Expose `window.getReduxState()` to make snapshots to storybook easy
+if (typeof window !== 'undefined') {
+    // Disabled in production to prevent leaking secret data, personal API keys, etc
+    if (process.env.NODE_ENV === 'development') {
+        ;(window as any).getReduxState = () => getContext().store.getState()
```

Comment:
very cool

Reply:
This may makes sense in production as well, as you can already use the redux devtools to see the state. This just exposes it in a simple function.